AUTHORS.rst

```diff
@@ -1,6 +1,10 @@
 Contributors
 ============
 
+Please add your name using this markdown template:
+
+- `**authorname** <https://github.com/**authorname**>`_ | `contributors <https://github.com/ericmjl/pyjanitor/pulls?utf8=%E2%9C%93&q=is%3Apr+author%3A**authorname**>`_
```

Comment:
I think a code block might be better here.

https://www.sphinx-doc.org/en/1.5/markup/code.html

No need to mark what language is used.